httpbin/responses.go

```diff
@@ -13,6 +13,10 @@ const (
 	textContentType   = "text/plain; charset=utf-8"
 )
 
+type environmentResponse struct {
+	Environment map[string]string `json:"env"`
```

Comment:
Naming nit:  The endpoint is `/env` and the key in the JSON response is `"env"`, let's call this `envResponse` and name the struct member `Env` to match?

Reply:
Sure, done